common/lib/dependabot/pull_request_creator/codecommit.rb

```diff
@@ -49,6 +49,7 @@ def base_commit_is_up_to_date?
       def create_pull_request
         branch = create_or_get_branch(base_commit)
         return unless branch
+        create_commit
```

Comment:
Looks like rubocop is complaining about this

```suggestion

        create_commit
```

Reply:
fixed!